docs/modules/tiered-storage/pages/configuration.adoc

```diff
@@ -0,0 +1,179 @@
+= Configuration
```

Comment:
```suggestion
= Configuring Tiered Storage
```

The title should contain the keywords that will help users find the page and understand what's on it.